src/serializable.h

```diff
@@ -0,0 +1,17 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+
+#pragma once
+#include "types.h"
+#include "core/streaming.h"
+
+namespace librealsense
+{
+    class serializable_interface
```

Comment:
If the class is 'serializable_interface', IMO the file-name should be the same.

This is meant to be implemented by devices only, right? (I assume that's why the extension is 'SERIALIZABLE_DEVICE'?) Should the name reflect that?